github/GithubObject.py

```diff
@@ -92,6 +93,26 @@ def remove_unset_items(data: Dict[str, Any]) -> Dict[str, Any]:
 Opt = Union[T, _NotSetType]
 
 
+def is_defined(v: Union[T, _NotSetType]) -> TypeGuard[T]:
+    return not isinstance(v, _NotSetType)
+
+
+def is_undefined(v: Any) -> TypeGuard[_NotSetType]:
```

Comment:
what about
```suggestion
def is_not_set(v: Any) -> TypeGuard[_NotSetType]:
```
given default values are `NotSet`.